trollmoves/dispatcher.py

```diff
@@ -259,7 +268,13 @@ def update_config(self, new_config):
                     # FIXME: make sure to get the last messages though
                     self.listener.stop()
                 self.config = new_config
-                self.listener = ListenerContainer(topics)
+                addresses = client_config.get('subscribe_addresses', None)
+                nameserver = client_config.get('nameserver', 'localhost')
+                services = client_config.get('subscribe_services', '')
```

Comment:
Don't we want to use the posttroll defaults here ? (instead of redefining them)

Reply:
Those are the default values `ListenerContainer` has. Don't know how to get the default values from `ListenerContainer` in the case they are not defined in the config by the user. Or is there a way to say `nameserver = client_config.get('nameserver', default_value_of_nameserver_in_ListenerContainer)` I'm unaware of?

Reply:
@mraspaud any additional comments/suggestions to this?

Reply:
well, we could go introspective, but I don't think it's necessary. This will do for now.